libs/portals/my-pages/social-insurance-maintenance/src/components/PaymentGroupTable/PaymentGroupTableRow.tsx

```
@@ -1,70 +1,209 @@
+          }}
+        >
+          <Text variant="medium" textAlign="right">
+            {`${amountFormat(paymentGroup.totalYearCumulativeAmount)}`}
+          </Text>
+        </T.Data>
+      </T.Row>
+      {paymentGroup?.payments.map((pg, idx) => {
+        const last = idx === paymentGroup.payments.length - 1
+
+        return (
+          <T.Row>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)